### ISSUE TITLE
Are missing blocks and transactions forwarded downstream?

### ISSUE THREAD

**Issue by linuskendall** [rpcpool/yellowstone-faithful#66]
If faithful doesn't have a block (i.e. the block is not among those available) does it forward getBlock downstream if the `--proxy` command is configured?

Same thing for a transaction that is not found? This is perhaps more dubious since a not found transaction might just mean it doesnt exist.

**Comment by gagliardetto** [comment]
> Same thing for a transaction that is not found? This is perhaps more dubious since a not found transaction might just mean it doesnt exist.

Yes. Just to be sure, anything that isn't a success, is considered a failure and retried on the fallback rpc.

**Comment by anjor** [comment]
Sounds like this is not needed anymore. Can we close it @linuskendall @gagliardetto?
